README.md

```diff
@@ -152,7 +173,7 @@ merge:
       # If "body" is "pull_request_body", then the commit message will be the
       # part of the pull request body surrounded by "message_delimiter"
       # strings. This is disabled (empty string) by default.
-      message_delimiter: ==COMMIT_MSG==
+    message_delimiter: ==COMMIT_MSG==
```

Comment:
Was this indentation actually incorrect?

Reply:
That must have been a mistake, good catch